example/pythonlib/basic/3-multi-module/build.mill

```diff
@@ -0,0 +1,39 @@
+//// SNIPPET:BUILD
```

Comment:
We don't need these snippet comments here since we're not sharing example code with the other languages